xcell/cls/cl.py

```diff
@@ -293,7 +394,10 @@ def get_cl_file(self):
         ell = np.arange(3 * nside)
         if not os.path.isfile(fname):
             ccl_tr1, ccl_tr2 = self.get_tracers_ccl()
-            cl = ccl.angular_cl(self.cosmo, ccl_tr1, ccl_tr2, ell)
+            cl = self._get_ccl_cl(ccl_tr1, ccl_tr2, ell)
+            b1 = self.data.data['tracers'][self.tr1].get('bias', 1.)
```

Comment:
Should we add here the `if self.data['cov']['fiducial']['gc_bias']` or do you think we should always add the galaxy bias (which I guess 99% we will want that)?

Reply:
I convinced myself that the `if` is not necessary. If you don't want to account for the galaxy bias, just remove it from the tracers or set it to 1.